### ISSUE TITLE
H1 tag should be used on both home and front-page cases

### ISSUE THREAD

**Issue by audrasjb** [WordPress/twentytwenty#23]
H1 tag should ideally be used on both home and front-page cases.

**Comment by joyously** [comment]
Does it make sense that the footer widget areas are H2?

**Comment by joyously** [comment]
My opinion of how the H1 should be:
- static home Page (`is_front_page() && ! is_home()`) - site title is H1, no page title
- default Latest Posts (`is_front_page() && is_home()`) - site title is H1, no page title
- normal static Page (`! is_front_page() && ! is_home()`) - site title is div, page title is H1
- Latest Posts static Page (`! is_front_page() && is_home()`) - site title is div, page title is H1

Isn't this what the original code was doing?

**Comment by afercia** [comment]
Worth also noting that when the homepage is set to a static page, there are two H1 headings. Whatever the choice is, this should be avoided and the heading hierarchy should not skip levels. It should not rely on the content authors will use on the page.

<img width="1428" alt="Screenshot 2019-09-07 at 13 14 43" src="https://user-images.githubusercontent.com/1682452/64474186-c0f06380-d171-11e9-94dc-4cafa6dec38c.png">
